src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -2361,12 +2361,22 @@ public static string LastWriteTimeString(PSObject instance)
                     // non-existing targets on either Windows or Linux.
```

Comment:
@iSazonov, your last commit had 2 failures in `PowerShell-CI-windows`
New-Item: symlink with absolute/relative path test.Symlink with relative path to existing directory behaves like a directory
```powershell
Expected the value to have type [System.IO.DirectoryInfo] or any of its subtypes, but got $null with type $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\New-Item.Tests.ps1: line 305
305:         Get-Item someDirLinkRelative | Should -BeOfType System.IO.DirectoryInfo
```
New-Item: symlink with absolute/relative path test.Symlink with absolute path to existing file behaves like a file
```powershell
someFileLinkAbsolute
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\New-Item.Tests.ps1: line 309
309:         New-Item -Type SymbolicLink someFileLinkAbsolute -Target (Convert-Path someFile)

```